src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Controllers/ImportControllerTests.cs

```diff
@@ -100,6 +100,7 @@ private ImportController GetController(ImportTaskConfiguration bulkImportConfig)
                 _urlResolver,
                 optionsOperationConfiguration,
                 optionsFeatures,
+                null,
```

Comment:
We can replace null with Substitute.For<IResourceWrapperFactory resourceWrapperFactory>()

Reply:
Good idea